cpp/tests/reductions/reduction_tests.cpp

```diff
@@ -135,7 +160,9 @@ TYPED_TEST(MinMaxReductionTest, MinMax)
   std::vector<int> int_values({5, 0, -120, -111, 0, 64, 63, 99, 123, -16});
   std::vector<bool> host_bools({1, 1, 0, 1, 1, 1, 0, 1, 0, 1});
   std::vector<bool> all_null({0, 0, 0, 0, 0, 0, 0, 0, 0, 0});
-  std::vector<T> v = convert_values<T>(int_values);
+  std::vector<T> v       = convert_values<T>(int_values);
+  T init_value           = convert_int<T>(321);
```

Comment:
The value `321` is too large. Imagine that if the type test is `uint8_t` then this will overflow and you may get undefined behavior. So, it is safe to always use numbers in the range `[0, 100]` (`[0, 127]` is still safe, but `[0, 100]` is easier to remember).

Reply:
Unless you explicitly specify a type (like `int32_t`, in a `TEST_F`), never use negative values in a typed test.